tests/src/Interop/common/xplatform.h

```diff
@@ -57,7 +57,7 @@
 #define WINAPI __stdcall
 #endif
```

Comment:
There is no practical difference in non-windows platform for stdcall vs _cdecl (and we get to define what WINAPI means in non-windows platform - and it is just __stdcall, whatever that means). What I've fixed last time is to avoid always #define WINAPI   to _cdecl, which is obviously incorrect in windows.